docs/docsite/rst/community/steering/steering_committee_membership.rst

```diff
@@ -50,6 +50,10 @@ After the announcement is made, the new representative is added to the `Steering
 
 The team uses the same Community Topic for announcing subsequent representative changes. Representatives should commit to at least two months of membership.
 
+The team representative must still abide by all expectations listed in :ref:`steering_expectations`, including those surrounding participation.
+Steering Committee members are generally expected to participate in discussions — synchronously and/or asynchronously — and votes,
```

Comment:
What do folks think about this clarification?

```suggestion
Steering Committee members are generally expected to participate in discussions — synchronously in meetings and/or asynchronously on the forum — and votes,
```

Reply:
SGTM, though I would put an emphasis on the async way: maybe would just put it first, then the sync one.

Reply:
See the latest push